lib/markdown.tsx

```diff
@@ -14,20 +14,45 @@ interface Response {
     ];
   };
   responses: {
-    [key: string]: string;
+    [key: string]: string | [];
   };
 }
 export default (formResponse: Response): string => {
   const formElements = formResponse.form.elements;
   const responses = formResponse.responses;
   const title = `${formResponse.form.titleEn} / ${formResponse.form.titleFr}`;
 
-  const mdBody = formElements.map((element, index) => {
+  const mdBody = formElements.map((element) => {
     // In future add validation to remove page elements
     const qTitle = element.properties.titleEn;
-    return {
-      p: `${qTitle}${String.fromCharCode(13)} ${responses[element.id]}`,
-    };
+    if (Array.isArray(responses[element.id])) {
+      if (responses[element.id].length) {
+        const options = (responses[element.id] as Array<string>)
+          .map((option: string) => {
+            return `- ${option}`;
+          })
+          .join(String.fromCharCode(13));
+
+        return {
+          p: `${qTitle}${String.fromCharCode(13)}${options}`,
```

Comment:
A tiny nitpick - do you think we can turn part of this into a `const` value as it appears a few times?